lib/comma/sanitized_extractor.rb

```diff
@@ -27,36 +27,35 @@ def __use__(style)
 
     private
 
+    # this is in case the result is only + and then numbers, as in the case of a phone number
     def check_for_only_digits(result)
       length = result.length
 
       result.start_with?("+") && (result.slice(1..length) !~ /\D/)
     end
 
-    def remove_special_characters_at_start(result)
-      while starts_with_special_characters(result)
-        result.slice!(0)
-      end
+    # results would then transform from "+astring" to "'+astring"
+    def append_result_with_apostrophe(result)
```

Comment:
Should this be named `prepend_result_with_apostrophe`?

Reply:
coding at 5 a.m. you forget what pre and app mean, respectively